### PR TITLE
Add missing watch for installation in auth controller

### PR DESCRIPTION
**Pull request by rene-dekker**
Add missing watch for installation in auth controller

### CODE REVIEW

**Review by Brian-McM — APPROVED** [review]

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Pull request merged**
